src/qtiItem/core/Element.js

```diff
@@ -187,11 +187,11 @@ var Element = Class.extend({
         if (typeof this.initObject === 'function') {
             append(this.getObject());
         }
-        _.each(this.metaData, function(v) {
+        _.each(this.metaData, function (v) {
```

Comment:
https://lodash.com/docs/4.17.15#forEach
```suggestion
        _.forEach(this.metaData, function (v) {
```